providers/song-controls-front.js

```diff
@@ -1,18 +1,43 @@
 const { ipcRenderer } = require("electron");
 
-let videoStream = document.querySelector(".video-stream");
+let video = document.querySelector("video");
+
 module.exports = () => {
-    ipcRenderer.on("playPause", () => {
-        if (!videoStream) {
-            videoStream = document.querySelector(".video-stream");
-        }
+    ipcRenderer.on("playPause", (_e, toPlay) => playPause(toPlay));
+};
+module.exports.playPause = playPause;
 
-        if (videoStream.paused) {
-            videoStream.play();
-        } else {
-            videoStream.yns_pause ?
-                videoStream.yns_pause() :
-                videoStream.pause();
+function playPause(toPlay = undefined) {
+    if (!checkVideo()) return;
+    
+    switch(toPlay) {
+        case undefined: 
```

Comment:
nit: could also be the default case for the switch, to be defensive

Reply:
done 👍🏼